src/PSR7/Validators/SerializedParameter.php

```diff
@@ -192,4 +198,23 @@ public function getSchema(): CebeSchema
     {
         return $this->schema;
     }
+
+    protected function getChildSchema(CebeSchema $schema, string $key): ?CebeSchema
+    {
+        if ($schema->type === CebeType::OBJECT) {
+            if ($schema->properties[$key] ?? false) {
+                return $schema->properties[$key];
+            }
+
+            if (! is_bool($schema->additionalProperties) && $schema->additionalProperties instanceof CebeSchema) {
```

Comment:
why `additionalProperties` equal to `true` is avoided here? This is also valid case with (in general) empty schema as a property spec (property can be anything)

Reply:
Yes. It is a valid case, but we can't detect the concrete schema of a child type to convert the value to it. But also we no need to do any convertation there if we allow anything.

Alll three variants of the schema will be valid if we request smth like `path?id[a]=1&id[b]=2` :

```
id:
  type: object
  additionalProperties: true
```
```
id:
  type: object
  additionalProperties:
    type: integer
```
```
id:
  type: object
  additionalProperties:
    type: string
```

BTW For now I think I can just remove this check - it not unnecessary because object type checking is more important.

Reply:
I think returning an `\stdClass` for `additionalProperties: true` as `id` value is purely valid here. Without any other guessing of property types

Reply:
In case `additionalProperties: true` we should return null because no child schema is presented. So values will be casts without any type in `SerializedParameter::castToSchemaType($value, $type)` method . E.q. value will be used as-is. This situation is still valid because we haven't type in the schema to compare the type of value with.

Reply:
Yea, null is processed as `any` indeed. This wasn't obvious from namings.